src/IObjectMakr.ts

```diff
@@ -63,6 +63,11 @@ export interface IObjectMakrSettings {
      * be run when made.
      */
     onMake?: string;
+
+    /**
+     * Optionally, a custom functions mapping for the classes can be passed in.
```

Comment:
Can you be a little more clear? The current explanation leads to questioning what is a custom functions mapping is, and why classes can/would be passed in.